rg_instructor_analytics/tasks.py

```diff
@@ -231,3 +237,12 @@ def grade_collector_stat():
             GradeStatistic.objects.update_or_create(**key_values)
 
         LastGradeStatUpdate(last_update=this_update_date).save()
+
+
+@task
+def run_common_static_collection():
+    """
+    Task for the initial data collection.
```

Comment:
s/Task for the initial data collection./The task for updating analytics data./

Reply:
Replace with `The task for updating analytics data.`. For your, variant flake8 generate error